packages/run-it/src/components/PerfTracker/perfTableUtils.tsx

```diff
@@ -41,63 +41,70 @@ export const tableColumns: ActionListColumns = [
     primaryKey: true,
     title: 'URL',
     type: 'string',
-    widthPercent: 18,
+    widthPercent: 15,
```

Comment:
one more column. gotta use the shrinkolator until we decide which columns to prioritize in the grid, or adjust the display with of the content pane.